### ISSUE TITLE
Window class does not match expected

### ISSUE THREAD

**Issue by QuentinMcCarthy** [ThioJoe/ThioJoe-AHK-Scripts#25]
It looks like something broke in my installation of this. I have it in my Program Files on my HDD, and I've tried running it as admin, but neither fix this issue.

I started up the executable today after realising it was failing to startup on boot (I was in the process of fixing that), but when I tried to use it, the window isn't appearing. I enabled debug and got these three lines:

```
Hotkey Pressed: ~Media_Play_Pause

Window ID: 47123948
Class: CabinetWClass

Window class does not match expected: CabinetWClass
```

I updated from the initial version I had - 1.1.0 - to 1.2.5 (latest) and the issue is still persisting.

**Comment by BestCoder1877** [comment]
# I Have The Same Issue

**Comment by ThioJoe** [comment]
Hm what version of Windows are you running and can you specify an example of what program it doesn't work with? There isn't really an installation, it's literally just one file, so there isn't really anything that can 'break' with the exe itself unless something with windows itself changed.

**Comment by ThioJoe** [comment]
Wait are you trying to use it on regular windows explorer? It's only meant to show up for 'Save As' or 'Open' type windows, not Windows Explorer in general

**Comment by QuentinMcCarthy** [comment]
> Wait are you trying to use it on regular windows explorer? It's only meant to show up for 'Save As' or 'Open' type windows, not Windows Explorer in general

Windows 11, I only have ExplorerPatcher, it worked fine when I first downloaded it (I say installation but I mean just download, lol), I've tried resetting the settings, I haven't tried removing the tilde.

What I mean by the window not appearing is I mean that pressing the hotkey doesn't bring up the context menu.

Also, yes I was trying to use it on regular Windows Explorer, I've completely forgotten what I was trying to do when I created this issue because I have a terrible memory (I might've been trying to move downloaded files). Thusly, also forgot this isn't meant to be used on Windows Explorer. My bad!
